spark/src/test/scala/org/apache/spark/sql/delta/skipping/ClusteredTableTestUtils.scala

```diff
@@ -60,6 +61,33 @@ trait ClusteredTableTestUtilsBase extends SparkFunSuite with SharedSparkSession
     assert(expectedClusteringColumns == actualClusteringColumns)
   }
 
+  // Verify the operation parameters of the last history event contains `clusterBy`.
+  protected def verifyDescribeHistoryOperationParameters(
+      table: String
+  ): Unit = {
+    val deltaLog = if (table.startsWith("tahoe.") || table.startsWith("delta.")) {
```

Comment:
is this a common pattern to validate path based?

Reply:
Yeah, I think so